base-misc/ca-certs/autobuild/build

```
@@ -18,5 +18,4 @@ install -dv "${SSLDIR}/certs"
 cp -v "$SRCDIR"/certs/* "${SSLDIR}/certs"
 
 abinfo "Creating a compatibility symlink for ca-certificates.crt ..."
-touch ${SSLDIR}/ca-bundle.crt
-ln -sfv ../ca-bundle.crt "${SSLDIR}/certs/ca-certificates.crt"
+ln -sv "${SSLDIR}/ca-bundle.crt" "${SSLDIR}/certs/ca-certificates.crt"
```

`${SSLDIR}` is an absolute path leading into acbs build directory. This symbolic link will be broken at runtime, which breaks SSL on almost every program relying on `/etc/ssl/certs/ca-certificates.crt`:

```
lrwxrwxrwx 1 root root    64  9月 24 14:37 ca-certificates.crt -> /var/cache/acbs/build/acbs.57gs2qwa/abdist/etc/ssl/ca-bundle.crt
```

I'd suggest change `${SSLDIR}/ca-bundle.crt` to `../ca-bundle.crt` instead.

---

Forgot about that, sorry.
```suggestion
ln -sv ../ca-bundle.crt "${SSLDIR}/certs/ca-certificates.crt"
```